static/usr/bin/core-desktop-session-wrapper.sh

```diff
@@ -61,6 +61,7 @@ fixup_xauthority &
 ln -sf "snap.$snap_name/wayland-0" $XDG_RUNTIME_DIR/wayland-0
 # Symlink sockets for pipewire and pipewire-pulse
 ln -sf "snap.pipewire/pipewire-0" $XDG_RUNTIME_DIR/pipewire-0
-ln -sf "snap.pipewire/pulse" $XDG_RUNTIME_DIR/pulse
+mkdir -p $XDG_RUNTIME_DIR/pulse
+ln -sf "../snap.pipewire/pulse/native" $XDG_RUNTIME_DIR/pulse/native
```

Comment:
Any reason to prefer this over just symlinking the parent directory?

Reply:
None, really... changing it.